cmd/swarm/upload_test.go

```diff
@@ -101,16 +121,207 @@ func testCLISwarmUp(toEncrypt bool, t *testing.T) {
 		if string(reply) != data {
 			t.Fatalf("expected HTTP body %q, got %q", data, reply)
 		}
+		log.Info("verifying uploaded file using `swarm down`")
+		//try to get the content with `swarm down`
+		tmpDownload, err := ioutil.TempDir("", "swarm-test")
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer os.Remove(tmpDownload)
+
+		bzzLocator := "bzz:/" + hash
+		flagss := []string{}
+		flagss = []string{
+			"--verbosity", "5",
+			"--bzzapi", cluster.Nodes[0].URL,
+			"down",
+			bzzLocator,
+			tmpDownload,
+		}
+		if toEncrypt {
+			flagss = []string{
+				"--verbosity", "5",
+				"--bzzapi", cluster.Nodes[0].URL,
+				"down",
+				bzzLocator,
+				tmpDownload}
```

Comment:
This list of flags is the same, should it include `--encrypt`?

Reply:
no, why should it? encrypt is transparent using double-length references

Reply:
Ok, so if it doesn't need an extra flag, then why have this `if toEncrypt` that just sets the exact same flags?

Reply:
correct. pushed an update